WAVLTree/src/WAVLTree.java

```diff
@@ -110,7 +127,9 @@ public String max()
     */
    public int size()
    {
-           return 42; // to be replaced by student code
+       WAVLNode root;
+       root = this.getRoot()
```

Comment:
Inside your class, you can directly access your own root (`this.root`, or simply `root`)

Reply:
gotcha :) thanks 👍